exp-player/addon/components/exp-free-response/template.hbs

```diff
@@ -2,6 +2,23 @@
   <p class="break-line">{{t 'survey.sections.2.instructions.firstSection'}}</p>
   {{#bs-form}}
     {{#bs-form-group}}
+      <label>{{t 'survey.sections.2.questions.14.label'}}</label><br>
+        {{#power-select
+          selected=time
+          options=times
+          placeholder=placeholder
+          searchEnabled=false
+          onchange=(action (mut time))
```

Comment:
Can you clarify how the pick-a-time question ties together?

1. The field doesn’t appear to be tied to a response field. Eg, it iterates over `times as time`, but when you select, it mutates `time`, rather than a value that could be tied to a specific response...
https://github.com/CenterForOpenScience/exp-addons/pull/178/files#diff-04664267214cdc73b1c7ad09d204cf3eR11
2. The answer field *seems* to be `responses.q4`, if just going by a new field is a guide, but its label is `survey.sections.2.questions.14.label`. This nomenclature seems confusing.
3. You have two time values in the select box- which representation actually gets persisted to the database? (we don’t want to be sending the entire 12/24hr JSON object to the DB field)